Documentation/Settings.md

```diff
@@ -39,11 +39,11 @@ This is the "on device help text".
 
 ### Setting: Power source
```

Comment:
Please follow the warning at the top of this file and _dont edit it directly_.
Instead edit the english translations and re-run the generation tooling to update this file.